### DISCUSSION TITLE
Still Big Probs with Lan and Wlan on RPI 4 (Solved)

### DISCUSSION THREAD

**Discussion by hjfisch** [rern/rAudio#1150] [Bug Reports]
Network is still dropping sometimes , so I need to power off and on .
Need to do this twice sometimes to get it back.
By powering up the Device in the mornig , I usually need need to do this
twice to get the Network working , but only with my RPI 4 .
Even with no Network the player plays from connected HDD.
This only happens with my RPI 4 and not with my RPI 3 .
If I use Volumio , my RPI 4 is working fine , but I like rAudio 1 , so I would like to stay with it .

**Comment by R101G** [comment]
@hjfisch  
I have no problem with a wired Pi4, but I have disabled all the on-board wireless in the boot/config.txt  file. I don't know if that makes a difference. 
If you are using wireless, what does the iwconfig command show? Are you using any USB3 devices with 2.4 wireless, or is the Pi4 using a 5GHz SSID? Is the region setting correct?

**Comment by hjfisch** [comment]
I use a 4tb HDD on USB 3 with extra power connection and a 5 ghz usb-wlan-stick . on-board wireless is disabled in config.txt and cmdline . Exactly the same with my RPI 3 , that is working without probs . If I switch to Volumio , the RPI 4 is also working without any problem . I think I will exchange the RPI 4 with a RPI 3 the next days.
B.t.w. I've had this problem for some time now , so I thought that's a problem of my RPI 4 .I've changed it
a couple of weeks ago , but the problem resists . Yesterday I've replaced the SD-Card with an USB-Stick ,
connected to the second USB 3 Port . No problem this mornig . The System starts the correct way . Will see
the next days , wether the problem is gone .

**Reply by R101G** [reply]
Have you got a different SSID for the 5GHz band? If not, what is stopping the system from switching between the bands? The USB stick might be acting as a shield - USB3 is known to cause wifi interference, and the Pi3 only has USB2. Have you tried only using the USB2 ports?

**Comment by hjfisch** [comment]
Same SSID for both bands . USB-Wlan-Stick is connected to USB 2 , HDD to USB 3 , USB-CD to USB 2 with extra power ,and now an USB-Boot-Stick to the second USB 3 port .
I'm not only loosing WIFI but Lan too . No Network at all sometimes . It is shown in my Router , but connection refused .
Same config on my RPI 3 with same USB-Wlan-Stick . Maybe there is a problem with the sd-card , but I've
tested a few . As I wrote before , the RPI 4 is now booting from an USB_Stick and I will see if this works the next days .

**Comment by R101G** [comment]
@hjfisch 
Logically, if you are using a wired network connection with wifi disabled, the only difference between the Pi3 and the Pi4 is the USB3 ports and the associated greater power demand. I suggest you try to get a basic wired setup working, rather than trying stuff at random. If the SD card works in the Pi3 it should work in the Pi4.

I don't understand how a wired DHCP connection can be refused. What does ifconfig show? Are there any voltage warnings?

Start with just a wired connection and the HDD plugged into a USB2 port, not a USB3. If that works reliably, change one thing at a time.

**Comment by hjfisch** [comment]
Changed RPI 4 to RPI 3 and probs are gone . A little bit slower , but fast enough for MPD . Also the RPI 3 stays a little cooler .
